staging/src/k8s.io/apiserver/pkg/admission/initializer/initializer.go

```diff
@@ -39,12 +40,14 @@ func New(
 	extInformers informers.SharedInformerFactory,
 	authz authorizer.Authorizer,
 	featureGates featuregate.FeatureGate,
+	stopCh <-chan struct{},
```

Comment:
@liggitt - I know there are a bunch of unit test failing now (due to a need to pass stopCh here in there). But before fixing them I would like to ensure that this approach is the desired one on how to implement it.

Reply:
hrmm, ideally this would be available at plugin init time, but that requires changing the signature of the plugin init function, which isn't great. asking the plugin to init things without a stop channel, then later request and replumb that down into everything it initialized is tricky